### PR TITLE
add docs for update procedure

### LINKED ISSUE #31: Provide update scenario for repo
As the repo is in steady development there needs to be a guide on how to stay up to date and also keep the changes one has done - either use stashing (probably too advanced) or simply consider renaming of files if experiments are made

### PR DESCRIPTION
**Pull request by capital-G**
fixes #31

### CODE REVIEW

File: docs/course-info/setup.rst [hunk 1]
```diff
@@ -273,3 +273,42 @@ while being in the root directory of the repository and click on the appearing l
...
+Updating and working with the repository
+----------------------------------------
+
+The course material is stored and distributed via `git <https://git-scm.com/>`__.
+As there are constantly updates on the course material there will be conflicts
+when we have worked locally on a notebook as *git* does not know how to resolve
+the local changes with the remote updates - this is commonly known as
+`merge conflicts <https://www.atlassian.com/git/tutorials/using-branches/merge-conflicts>`__.
+
+The best way is probably to save the notebook on which you work locally to a new
```
Comment (telephon) [inline docs/course-info/setup.rst:287]: More idiomatic: "Probably the best way is to … "

File: docs/course-info/setup.rst [hunk 2]
```diff
@@ -273,3 +273,42 @@ while being in the root directory of the repository and click on the appearing l
...
 
   Any help on this is appreciated.
+
+
+Updating and working with the repository
+----------------------------------------
+
+The course material is stored and distributed via `git <https://git-scm.com/>`__.
+As there are constantly updates on the course material there will be conflicts
+when we have worked locally on a notebook as *git* does not know how to resolve
```
Comment (telephon) [inline docs/course-info/setup.rst:283]: easier to parse:
… when we have worked locally on a notebook. This is because *git* …

**Review by telephon — CHANGES_REQUESTED** [review]
just minor polishing

**Comment by capital-G** [comment]
thanks for the suggestions
i implemented the changes
please merge if you think its OK

**Review by telephon — APPROVED** [review]

**Pull request merged**